tests/reconfiguration.py

```diff
@@ -212,11 +212,7 @@ def test_add_node_from_snapshot(network, args, copy_ledger=True, from_backup=Fal
     # is at least one historical entry to verify.
     network.txs.issue(network, number_txs=1)
     idx, historical_entry = network.txs.get_last_tx(priv=True)
-    for _ in range(1, args.snapshot_tx_interval):
-        network.txs.issue(network, number_txs=1, repeat=True)
-        last_tx = network.txs.get_last_tx(priv=True)
-        if network.wait_for_snapshot_committed_for(seqno=last_tx[1]["seqno"]):
```

Comment:
Removed the `wait_for_snapshot_committed_for` function which was duplicated in `get_committed_snapshots`.